gym/spaces/box.py

```diff
@@ -1,4 +1,5 @@
 import numpy as np
+from warnings import warn
```

Comment:
```suggestion
import warnings
```

It is better to match the style of the rest of the code-base.